README.md

```diff
@@ -19,69 +19,83 @@ How do I use it?
 
 You create a new EvaluableExpression, then call "Evaluate" on it.
 
+```go
 	expression, err := govaluate.NewEvaluableExpression("10 > 0");
-	result := expression.Evaluate(nil);
+	result, err := expression.Evaluate(nil);
```

Comment:
Good catch! I didn't notice that the readme didn't specify the right return values. :+1: